src/NUnitFramework/tests/TestContextTests.cs

```diff
@@ -335,9 +335,15 @@ public async Task TestContextOut_ShouldFlowWithAsyncExecution()
         [Test]
         public async Task TestContextWriteLine_ShouldNotThrow_WhenExecutedFromAsyncMethod()
         {
-            Assert.DoesNotThrow(TestContext.WriteLine);
+            Assert.DoesNotThrow(() => TestContext.WriteLine(string.Empty));
             await YieldAsync();
-            Assert.DoesNotThrow(TestContext.WriteLine);
+            Assert.DoesNotThrow(() => TestContext.WriteLine(string.Empty));
```

Comment:
This change should not be done. It is the result of removing a method.

Reply:
What does this code test?

Reply:
That is a whole other question.
It doesn't seem that any of the other `Write` methods are tested at all.